utils/hct/hctdb.py

```diff
@@ -1749,11 +1749,11 @@ def populate_dxil_operations(self):
 
         # End of DXIL 1.6 opcodes.
         self.set_op_count_for_version(1, 6, next_op_idx)
-        assert next_op_idx == 218, "next operation index is %d rather than 165 and thus opcodes are broken" % next_op_idx
+        assert next_op_idx == 218, "218 is expected next operation index but encountered %d and thus opcodes are broken" % next_op_idx
```

Comment:
Hah! I was going to fix this in an unrelated change. This wording is less confusing.